Cargo.toml

```
@@ -1,5 +1,6 @@
 [workspace]
 members = ["crates/*", "examples/*"]
+exclude = ["examples/ts-client/"]
```

```suggestion
exclude = ["examples/streams-ts/"]
```

Can we call it this because may have some none stream examples in the future.